energy-integration/common-modules/weekly-rewards-splitting/src/lib.rs

```diff
@@ -143,20 +156,23 @@ pub trait WeeklyRewardsSplittingModule:
         }
     }
 
+    // !!! TODO  - update user boosted rewards formula
     fn get_user_rewards_for_week(
```

Comment:
I agreed with what @dorin-iancu wrote. As only this function need to be rewritten in a separate special trait.